src/modules/api-keys.ts

```diff
@@ -31,28 +33,28 @@ export default class ApiKeys {
     endpoint: string,
     ethereumAddress: string,
     signingMethod: SigningMethod,
-    data?: {},
+    data: {} = {},
   ): Promise<Data> {
-    const requestPath: string = `/v3/${endpoint}`;
-    const timestamp: Date = new Date();
-    const signature: string = await this.signOffChainAction.signOffChainAction(
+    const requestPath = `/v3/${endpoint}`;
+    const timestamp = new Date().toISOString();
+    const body = JSON.stringify(data);
```

Comment:
tiny nit: types can be inferred here, but explicit types might help introducing of bugs in the future on accident since I think `axiosRequest.headers` will take any type?